jsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
 	"extends": "./.svelte-kit/tsconfig.json",
 	"compilerOptions": {
+		"ignoreDeprecations": "5.0",
```

Comment:
what does this do?

Reply:
Hide config file errors. Svelte uses old typescript config in `tsconfig.json` which causes any extended config file to have errors.

Reply:
cool cool